### ISSUE TITLE
Creating a Discord Server

### ISSUE THREAD

**Issue by pramakaisar** [lmg-anon/mikupad#83]
Have you guys thought about creating a discord server? I see mikupad has a lot of potential, and making communities and gaining support are easier in discord, especially for those who are new to github.

**Comment by lmg-anon** [comment]
Making a Discord server for this project feels very frivolous. What is there that needs discussion for a whole server to be made? The whole point of Mikupad is to be simple and straightforward. If you don't understand LLM parameters, then there are better places to learn about these things.

**Comment by pramakaisar** [comment]
> Making a Discord server for this project feels very frivolous. What is there that needs discussion for a whole server to be made? The whole point of Mikupad is to be simple and straightforward. If you don't understand LLM parameters, then there are better places to learn about these things.

Sharing presets, API / model recommendations, and a community overall. I couldn't see what could go wrong with making a discord servers. I've had a few of my friends still don't understand the text formatting on instruct models like ChatML and L3. Using mikupad is not as straightforward for beginners. Please do let me know your thoughts about this.

**Comment by neCo2** [comment]
In my opinion, public Discords serve little purpose beyond providing a place for the barely literate to ask the same 3 questions over and over again.
Not to mention, locking information behind a Discord account is a horrible idea for any number of reasons.

And as the maintainer said, there are better places to learn about how LLM prompting works a single Google search away.

**Comment by pramakaisar** [comment]
> In my opinion, public Discords serve little purpose beyond providing a place for the barely literate to ask the same 3 questions over and over again.
> Not to mention, locking information behind a Discord account is a horrible idea for any number of reasons.
> 
> And as the maintainer said, there are better places to learn about how LLM prompting works a single Google search away.

yep I agree on ur point. "serve little purpose beyond providing a place for the barely literate to ask the same 3 questions over and over again."

but it's the same illiterate people that are the target audience. If you look at NAI user base, the people still stay there because they wouldn't bother to search for other models even though some models are better. most writers lack the technical know how and would rather use something easy to use like NAI, but with adequate community support mikupad could be really popular.

Any increase of visibility is good. since popularity attracts developers and contributors to contribute to the project. if moderation is ur concern, you can delegate that to the community.

Do let me know ur guys opinion. Overall, Kudos to the developers for making this open source and available!

**Comment by lmg-anon** [comment]
Hm... I get what you mean. But, in my opinion, there's no need to take this action to attract more people to use Mikupad. Although more contributors would be nice, my only objective with Mikupad is to create a good text-completion frontend, not to build an LLM community. So I'm completely fine with how things are going right now. However, I won't stop you or anyone else from creating a Discord server on their own if they want to. It's not like I have the power to prevent it anyway!
